### DISCUSSION TITLE
Github Student approved but asking for reverification, but it's asking for wrong mail while reverifying.

### DISCUSSION THREAD

**Discussion by TenTraicion** [community/community#58216] [GitHub Education] [answered]
### Hi everyone,

My Student account was approved on Jun 11 and i got my github features on Jun 14 exactly after 72 hour.
![image](https://github.com/community/community/assets/103281314/b1b6c25b-0ef5-45c3-ba2a-6e1cf051c240)

but today on Jun 16 they are asking me to reverify: 
![image](https://github.com/community/community/assets/103281314/d5163047-44f4-4cac-95f3-02c047250c4f)
when i am reverifying my account using my student email, they are telling me to use a different mail instead of the mail i've provided by the school:
![image](https://github.com/community/community/assets/103281314/95ecba4d-936f-4f50-ab4f-481fb10d60cb)

here is the link of my school: https://www.nu.ac.bd/ and this is the notice from national university regarding email: https://www.nu.ac.bd/uploads/notices/notice_854_pub_date_07062023.pdf where it clearifies the mailing domain : mail@nu.ac.bd

what should i do to get the benefits?

**Comment by the-officialjosh** [comment]
The best solution is to send them this in their support email they should easily rectify this for you.

**Reply by TenTraicion** [reply]
what is there support mail? i can't find any.
tried to open a support ticket, but whenever i choose the subject (github student pack) it just redirects me to  https://education.github.com/discount_requests/application

![image](https://user-images.githubusercontent.com/103281314/232054197-803ffdf8-18cd-488e-becb-48daea363490.png)
![image](https://user-images.githubusercontent.com/103281314/232054278-b638148d-a5e7-4feb-bbab-4fda06b85883.png)

**Comment by the-officialjosh** [comment]
Try out from here - https://support.github.com/request/education?tags=dotcom-direct%2Chubberfy_education

**Comment by TenTraicion** [comment]
4 Days and no reply
![image](https://github.com/community/community/assets/103281314/8959accd-ddd1-4eb4-ac34-1ff2686fc2ac)

also can't use my email either:
![image](https://github.com/community/community/assets/103281314/5cd417a5-1f3d-434e-a07d-7fa0cd0dddb0)

**Comment by TenTraicion** [comment]
Lost interest on github students program. they can just remove the school if they don't want to give this benefits
![image](https://github.com/community/community/assets/103281314/3795403e-1f6d-44e1-b798-e593ea1f30f2)

**Comment by samus-aran** [comment]
Hi @TenTraicion 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by TenTraicion** [reply]
hello there, thanks for noticing. it's been almost 2 months. Is there any update regarding the issue? It will be a great help if this is solved asap.

**Comment by dotjson01** [comment]
Same this Case Happened with me !
I am Reverifying my account which i earlier applied from but now this time won't taking this mail even my semester has'nt finished , I am still a college student . 

Kindly also look towards this !

![Screenshot 2023-08-02 004250](https://github.com/community/community/assets/84167034/45d9b491-fc6c-49f5-9f6b-f42f88cd4c3f)

**Comment by TenTraicion** [comment, marked as answer]
Github fixed the issue for me. Thanks a lot for that. Now it automatically selects the school by fetching my Email. That's a really good Update.
